src/logcollector/logcollector.c

```diff
@@ -651,12 +651,12 @@ void LogCollectorStart()
 
                         char msg_alert[512 + 1];
 
-                        snprintf(msg_alert, 512, "ossec: File rotated (inode "
+                        snprintf(msg_alert, 512, "wazuh: File rotated (inode "
                                  "changed): '%s'.",
                                  current->file);
 
                         /* Send message about log rotated */
-                        w_msg_hash_queues_push(msg_alert, "wazuh-logcollector", strlen(msg_alert) + 1, default_target, LOCALFILE_MQ);
+                        w_msg_hash_queues_push(msg_alert, "ossec-logcollector", strlen(msg_alert) + 1, default_target, LOCALFILE_MQ);
```

Comment:
Here, we are replacing "**wazuh**-logcollector" by "**ossec**-logcollector"

Reply:
It's true! This was a rebase error. Nice catch.